### PR TITLE
fixes: dashboard

### PR DESCRIPTION
**Pull request by Codebox124**
## Related Issue(s)

Fixes #
Resolves #
Closes #

## Proposed Changes

[//]: # '[List down the specific changes you have made in this pull request.]'

- [x]

## Checklist

- [x] I have followed the repository's contribution guidelines.
- [x] I have added necessary documentation or updated existing documentation.
- [x] My code follows the established code style and formatting of the project.
- [x] I have tested my changes and they function as expected.
- [ ] I have added suitable unit tests (if applicable).

## To-Do

- [ ]

### CODE REVIEW

**Review by techyNonso — CHANGES_REQUESTED** [review]
@Nasfame  please lookn into this builld error, it is outside the scope of @Codebox124 task. It has to do with prisma.
Also look at file conflicts

**Review by Nasfame — COMMENTED** [review]
lgtm

**Comment by Nasfame** [comment]
Resolves #39

**Comment by Nasfame** [comment]
@Codebox124 still not fixed : https://realfavicongenerator.net/favicon_checker?protocol=https&site=app-decenterai-com-git-fixes-favicon-decenter-ai.vercel.app

**Comment by Nasfame** [comment]
@Codebox124 update the pr

**Comment by Nasfame** [comment]
there are unwanted changes, I guess

File: src/app/dashboard/repository/[id]/components/modal.tsx [hunk 1]
```diff
@@ -1,59 +0,0 @@
-'use client'
```
Comment (Nasfame) [inline src/app/dashboard/repository/[id]/components/modal.tsx:1]: keep the code, leave doc comment stating the issue u were facing

**Review by Nasfame — COMMENTED** [review]
...

**Review by Nasfame — APPROVED** [review]
LGTM

**Pull request merged**